pkg/tsdb/cloudwatch/metric_find_query.go

```diff
@@ -585,6 +647,28 @@ func (e *CloudWatchExecutor) ec2DescribeInstances(region string, filters []*ec2.
 	return &resp, nil
 }
 
+func (e *CloudWatchExecutor) resourceGroupsGetResources(region string, filters []*resourcegroupstaggingapi.TagFilter, resourceTypes []*string) (*resourcegroupstaggingapi.GetResourcesOutput, error) {
```

Comment:
Would like some tests of this to validate tjat expected input/output can be handled correctly.

Reply:
I did not add an explicit test for this method, as it is merely a helper function that combines paged AWS results into one big response and already gained full coverage through the test for handleGetResourceArns